ps1/ps1/src/twitter/SocialNetwork.java

```diff
@@ -1,134 +0,0 @@
-     *            Twitter usernames in the returned social network must be
-     *            either authors or @-mentions in the list of tweets.
-     */
-    public static Map<String, Set<String>> guessFollowsGraph(List<Tweet> tweets) {
-        
-        Map<String, Set<String>> userNetwork = new HashMap<>();
-        
-        // get all mentioned users
-        Set<String> userList = Extract.getMentionedUsers(tweets);
-        
-        // add authors who weren't mentioned to this list
-        // to get total user list
-        for (int i = 0; i < tweets.size(); i++){
-            if (!userList.contains(tweets.get(i).getAuthor().toLowerCase())){
-                userList.add(tweets.get(i).getAuthor().toLowerCase());
-            } else {
-                continue;
-            }
-        }
-        
-        // now cycle through to find tweets authored or @-mentioned
-        for (String user : userList) {
-            
-            user = user.toLowerCase();
-                     
-            // set of users mentioned in own tweets
-            List<Tweet> ownTweets = Filter.writtenBy(tweets, user);
-            Set<String> Network = Extract.getMentionedUsers(ownTweets);
-            
-            // set of authors mentioning user in their tweets
-            // error here: you need to get ALL tweets where they're mentioned,
-            // inclduing those that are uppercase. Fix this.
-            List<Tweet> mentionedTweets = Filter.containing(
-                    tweets, Arrays.asList("@".concat(user)));
-            
-            for (int i = 0; i < mentionedTweets.size(); i++) {
-                if(!Network.contains(mentionedTweets.get(i).getAuthor().toLowerCase())) {
-                    Network.add(mentionedTweets.get(i).getAuthor().toLowerCase());
-                    
-                } else {
-                    continue;
-                }
-            }
-            
-            // remove username if present
-            Network.remove(user);
-            
-            userNetwork.put(user, Network);
-            
-        }
-        
-        return userNetwork;
-        
-    }
-
-    /**
-     * Find the people in a social network who have the greatest influence, in
-     * the sense that they have the most followers.
-     * 
-     * @param followsGraph
-     *            a social network (as defined above)
-     * @return a list of all distinct Twitter usernames in followsGraph, in
-     *         descending order of follower count.
-     */
-    public static List<String> influencers(Map<String, Set<String>> followsGraph) {
-        
-        Map<String, Integer> numberFollowers = new HashMap<>();
-        
-        for(String user : followsGraph.keySet()) {
-            
-            numberFollowers.put(user, followsGraph.get(user).size());
-            
-        }
-        
-        // sort a map by size
-        // stolen from stack exchange
```

Comment:
haha yes, we got the same I think ! That one was tough, and it was the end of the problem set.... was tired